src/Relation/Binary/Properties/Setoid.agda

```diff
@@ -77,6 +79,15 @@ preorder = record
 ≉-resp₂ : _≉_ Respects₂ _≈_
 ≉-resp₂ = ≉-respʳ , ≉-respˡ
 
+------------------------------------------------------------------------
+-- Equality is closed under composition
+
+≈;≈⇒≈ : _≈_ ; _≈_ ⇒ _≈_
+≈;≈⇒≈ (_ , p , q) = trans p q
+
+≈⇒≈;≈ : _≈_ ⇒ _≈_ ; _≈_
+≈⇒≈;≈ q = _ , q , refl
+
```

Comment:
I would be _strongly_ tempted to refactor these proofs, either in this PR, or subsequently, in terms of properties of general binary relations:
* any reflexive relation `R` (of the correct type) satisfies `S ⇒ R ; S` and `T ⇒ T ; R`;
* any transitive relation `T` satisfies `T ; T ⇒ T`;
* `Setoid`/`IsEquivalence`, `Preorder/`IsPreorder` satisfy reflexivity and transitivity: QED

Specialising to `Setoid` here seems ... a missed opportunity.

Reply:
My aim was to avoid having to come up with names for these :sweat_smile:

Reply:
Ouch!

Reply:
Seems a shame to hold up this PR for this refactor - though it sure would be nice to do the refactor before the release so that no deprecations are needed.

Reply:
Ok, let's merge, and work on the refactor later.